src/store/definition/reducer.js

```diff
@@ -1,11 +1,207 @@
-let initialState = null;
+import { ActionType } from './constants';
+
+import Immutable from 'immutable';
+
+import RequestRecord from '../../records/RequestRecord';
+import PropertyRecord from '../../records/PropertyRecord';
+import ResponseRecord from '../../records/ResponseRecord';
+
+const definition = new Immutable.Map({
```

Comment:
What's the purpose in defining this const? It doesn't look like it's used anywhere?